agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -586,7 +586,8 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
             LOG(DEBUG) << "Start ACS";
 
         // Set AP channel
-        if (!ap_wlan_hal->switch_channel(request->cs_params().channel,
+        if (ap_wlan_hal->get_radio_info().channel != request->cs_params().channel &&
```

Comment:
I'm really not sure about this one - from what I can remember, we need to trigger ACS by calling switch channel.
@vitalybu @morantr please comment.
@YuriyMasechko Is this commit absolutely needed? It looks like an optimization.

Reply:
It's optimization for case if only tx power limit was received and no channel switch is required. In such case we just path current channel to ap manager.